src/test/FuzzerImpl.cpp

```diff
@@ -537,21 +714,21 @@ TransactionFuzzer::initialize()
         xdr::xvector<Operation> ops;
         for (uint8_t i = 0; i < FuzzUtils::NUMBER_OF_PREGENERATED_ACCOUNTS; ++i)
         {
+            if (ops.size() > MAX_OPS_PER_TX - 1)
+            {
+                attemptToApplyOps(ltx, root.getPublicKey(), ops, *mApp);
+                ops.clear();
+            }
+
             PublicKey publicKey;
             FuzzUtils::setShortKey(publicKey, i);
 
             auto createOp = txtest::createAccount(
                 publicKey,
-                FuzzUtils::INITIAL_LUMEN_AND_ASSET_BALANCE * 10000000);
+                FuzzUtils::initialAccountBalance(accountParameters[i]));
```

Comment:
I can't help but feel like you are doing a lot of work here that you don't need to do. It's hard for me to even review this PR as it stands because I think that if we were to simplify what we are doing here, most of this PR wouldn't even exist.

Here's my core observation: you aren't computing the initial trust line balance, so why are you computing the initial account balance?

I think that there is a pretty elegant way to back into the desired outcome. For each account A:
1. Create A with an unnecessarily high balance (eg. 1M native, even though reserves and liabilities are in the range 0-100).
2. Create trust lines, offers, other ledger entries for A
3. Use `getMinBalance` or `getAvailableBalance` to determine the amount of excess funds
4. Pay most of the excess funds back to the root, leaving `DEFAULT_STROOPS_AVAILABLE_FOR_TEST_ACTIVITY` behind

The advantages of such an approach are primarily:
1. You can do whatever set-up you want without much risk of failure; if it does fail, you'll find out immediately when you try to run it anyway
2. You don't need to guess the balance in advance which is requiring you to do a lot of tracking and run-time computation
3. You are outsourcing all computation to the internals of stellar-core, so you don't need to depend on explicitly computing liabilities or minimum balances. This will make the solution more robust to future protocol changes

Would this work? Or am I missing something here?

Reply:
It sounds plausible to me.  Besides paying back excess funds after setup, should we also start with high trust line limits and reduce them after setup, for symmetrical reasons?

Reply:
Keep it simple first. Let's see what happens if you do this for accounts, and we can decide if we want to do it for trust lines too.